ocfweb/about/templates/about/staff.html

```diff
@@ -82,13 +82,14 @@ <h3>Computer Lab &amp; Printing</h3>
     <div class="ocf-lab">
         <div class="container">
             <div class="row">
-                <div class="col-md-7">
-                    <img src="{% static 'img/about/server-lab.png' %}" alt="servers and lab" />
+                <div class="col-md-7" style="padding-top:3em;">
```

Comment:
You could also consider `margin: auto 0;`, which might get closer to what you want (vertical centering). This is definitely an improvement though. If you want to merge as-is, that's fine with me.

Reply:
Also, if you place it in a separate CSS file, you might be able to use media queries to remove the margin/padding on mobile.

Reply:
> `margin: auto 0;`
unfortunately that does not help in this case as the `div` element of the column automatically is the size of the picture (the only element in that `div`), hence that `div` does not expand to the full height of the column next to it.

Reply:
ah, makes sense. I'd go ahead and merge

Reply:
moved inline css to the proper css file; the webpage looks the same as previous commit.